crates/kitsune_p2p/timestamp/src/human.rs

```diff
@@ -8,7 +8,9 @@ use std::convert::TryFrom;
 #[derive(Clone, Copy, Debug, Deserialize, Serialize)]
 #[serde(untagged)]
 pub enum HumanTimestamp {
+    /// A microsecond resolution [`Timestamp`].
     Micros(Timestamp),
+    /// A RFC3339 [`DateTime`].
     RFC3339(DateTime),
```

Comment:
o_O - the `DateTime` alias here is defined in chrono_ext as `pub(crate)` -- i assume this is allowed because it gets redirected to the actual type in the chrono lib, but it's a weird indirection. Also, I'm not sure I understand the name RFC3339 here, since it's a generic datetime type, which can possibly be represented by rfc3339...

Reply:
This is a bit off topic for this PR. @abe-njama we should make a separate story for renaming this.

Reply:
@freesig I've created a separate story for this; [B-04961](https://www51.v1host.com/Holo/story.mvc/Summary?oidToken=Story:41708)